libgd/plan.sh

```diff
@@ -0,0 +1,113 @@
+pkg_name=libgd
+pkg_origin=core
+pkg_version="2.2.4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=$(cat << EOF
```

Comment:
`pkg_license` is normally an array. Not sure but you might need an extra set of parens around this.